tests/helpers_tests/test_validator.py

```diff
@@ -146,6 +146,25 @@ def test_is_non_empty_string(self):
     self.assertTrue(validator.is_non_empty_string('0'))
     self.assertTrue(validator.is_non_empty_string('test_user'))
 
+  def test_is_attribute_valid(self):
+    """ Test that non-string attribute key or unsupported attribute value returns False."""
+
```

Comment:
nit. Just annotate that here you are testing for invalid attribute keys by putting a comment. Similarly for the next 2 sections.